src/components/ScholarshipAmountField.jsx

```diff
@@ -104,8 +111,7 @@ ScholarshipAmountField.propTypes = {
   helperText: PropTypes.string,
 };
 ScholarshipAmountField.defaultProps = {
-  className: null,
-  amountType: null,
+  amountType: '',
```

Comment:
It's not a string type.
```suggestion
  amountType: undefined,
```